### PR TITLE
Issue-372  Clarify PMIX_CMD_LINE in spawn case.

### PR DESCRIPTION
**Pull request by dsolt**
Signed-of-by:  dsolt@us.ibm.com

### CODE REVIEW

**Review by rhc54 — APPROVED** [review]
Sounds right to me!

File: Chap_API_Reserved_Keys.tex [hunk 1]
```diff
@@ -253,6 +253,9 @@ \subsection{Job realm attributes}
 %
 \declareAttributeNEW{PMIX_CMD_LINE}{"pmix.cmd.line"}{char*}{
 Command line used to execute the specified job (e.g., "mpirun -n 2 --map-by foo ./myapp : -n 4 ./myapp2").
+If the job was created by a call to \refapi{PMIx_Spawn}, the string is formed by concatenating together
+in order the values of \refattr{PMIX_APP_ARGV} for each application in the 
+job using the character ':' as a separator.
```
Comment (dsolt) [inline Chap_API_Reserved_Keys.tex:258]: The string is an in-order concatenation of the values of ....

**Comment by jjhursey** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

* You do not need to vote on every proposal
* If you have no opinion, don't vote - that is also useful data
* If you've already commented on this issue, please still vote so
      we know your current thoughts
* Not all proposals solve exactly the same problem, so we may end
      up accepting proposals that appear to have some overlap
This is not a binding majority-rule vote, but it will be a very
significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

* Hooray or Rocket: I support this so strongly that I
      want to be an advocate for it
* Heart: I think this is an ideal solution
* Thumbs up: I'd be happy with this solution
* Confused: I'd rather we not do this, but I can tolerate it
* Thumbs down: I'd be actively unhappy, and may even consider
      other technologies instead
If you want to explain in more detail, feel free to add another
comment, but please also vote on this comment.

**Comment by jjhursey** [comment]
PMIx ASC 1Q 2022

 * Passed the first vote: 11 yes / 0 no / 0 abstain
 * This is can be merged into main/v5 and v4 branches (I'll work on that)

**Review by jjhursey — DISMISSED** [review]

**Comment by jjhursey** [comment]
**Editorial**: Rebased the reference branch without text changes to prepare to merge.

**Review by jjhursey — APPROVED** [review]

**Pull request merged**